data/hi/retired/Stacelynn-KM-Eli-80fb08bd-6518-4109-bf2e-5915fe78bbb7.yml

```diff
@@ -7,14 +7,10 @@ image: http://capitol.hawaii.gov/Members/Images/RepSenPhotos/eli.jpg
 party:
 - name: Democratic
 roles:
-- type: lower
+- end_date: '2022-12-30'
```

Comment:
end date: 11/8/2022

Reply:
@samanthacara you can ignore HI for now these were fixed on another PR